internal/common/types.go

```diff
@@ -126,13 +126,13 @@ type Config struct {
 	// Logging contains logging-specific configuration settings.
 	Logging LoggingInfo
 	// Schedules is created on startup.
-	Schedules []models.Schedule
+	Schedules []models.Schedule `consul:"-"`
```

Comment:
What is the purpose of adding `consul:"-"`

Reply:
the pre-defined Devices, Schedules, and Schedule Events won't populate and read from Consul.
Also, Consul doesn't support array type.
``consul:"-"` `could let the Consul libraries ignore this.